lib/rspec/rails/example/rails_example_group.rb

```diff
@@ -11,7 +11,7 @@ module RailsExampleGroup
       include RSpec::Rails::MinitestLifecycleAdapter if ::Rails::VERSION::STRING >= '4'
       include RSpec::Rails::MinitestAssertionAdapter
       include RSpec::Rails::Matchers
-      include RSpec::Rails::FixtureSupport
+      include RSpec::Rails::FixtureSupport if RSpec::Rails::FixtureSupport rescue false 
```

Comment:
We actually specifically check for `if defined?(ActiveRecord::TestFixtures)` and make this module a no op if so.